scripts/genmap.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Same here, please add Apache 2.0 license header, attributed to Amazon (year 2020).